End-to-end support for coarser-than-daily count signals (#794)
CDCgov/PyRenew: docs/tutorials/building_multisignal_models.qmd

subpop_rt_deviation_process = RandomWalk(innovation_sd=0.025)
```

### Choosing the Rt Parameter Cadence

The renewal equation is evaluated on the model's daily time axis, but the temporal process for $\mathcal{R}(t)$ does not have to sample a new parameter every day.
This separates three model choices:

-   **Parameter cadence**: how often the $\mathcal{R}(t)$ temporal process samples a new latent value
-   **Model time axis**: the daily axis used by the renewal equation and delay convolutions

> -   **Observation cadence**: the scale of the likelihood for each signal, such as daily ED visits or weekly hospital admissions

Reviewer: ```suggestion
-   **Observation cadence**: the temporal granularity each signal, such as daily ED visits or weekly hospital admissions
```

The AR(1) process above samples one value per model day:

```python
baseline_rt_process = AR1(autoreg=0.9, innovation_sd=0.05)
```

> To parameterize $\mathcal{R}(t)$ weekly while still running the renewal equation daily, wrap the temporal process in `StepwiseTemporalProcess`.

Reviewer: ```suggestion
To use a weekly $\mathcal{R}(t)$ while still running the renewal equation daily, wrap the temporal process in `StepwiseTemporalProcess`.
```

The wrapper samples a coarse trajectory and broadcasts it to the daily model axis before the latent infection process uses it.

```python
weekly_baseline_rt_process = StepwiseTemporalProcess(
    inner=AR1(autoreg=0.9, innovation_sd=0.05),
    step_size=7,
    alignment="calendar_week",
    week=MMWR_WEEK,
)
```

Use `alignment="calendar_week"` when the weekly Rt blocks should align to a calendar week.

> Pass the same `WeekCycle` to every weekly component that must agree on the calendar: the `StepwiseTemporalProcess` above and any weekly `PopulationCounts` or `SubpopulationCounts` observation.

Reviewer: This is the first time `WeekCycle` is mentioned in the tutorial. Needs to be introduced a bit sooner.

Author: introduced.

`pyrenew.time` exports `MMWR_WEEK` (Sunday-Saturday epiweeks) and `ISO_WEEK` (Monday-Sunday); use `WeekCycle(start_dow=k)` for any other convention.
At sample or run time, pass the date of the first observation day as `obs_start_date`.
The model handles the calendar bookkeeping and forwards the day-of-week information to every component that needs it:

```python
model.run(
    num_warmup=500,
    num_samples=1000,
    rng_key=random.PRNGKey(42),
    n_days_post_init=n_days_post_init,
    population_size=population_size,
    obs_start_date=date(2024, 1, 7),

> 

Reviewer: No actual observations were passed here. Also extra line.

Author: removing code snippet - arguments to `run` are shown later in the tutorial.

    n_days_post_init=n_days,
    population_size=population_size,
    subpop_fractions=subpop_fractions,
    **obs_data,
)
samples = model.mcmc.get_samples()
```

where `**obs_data` is a data dictionary which supplies the observation start date as `obs_start_date`  and a data dictionary for each set of signal data, where the name of the data dictionary corresponds to the signal name registered on the builder.

> 

Reviewer: Write out an should write out the `obs_data` for this example.

Reviewer: (now that we no longer have the examples from the keyword `hospital=` and `wastewater=` pattern

Author: wrote out dict structure, keys only.
